tests/integration/helpers.py

```diff
@@ -299,12 +320,27 @@ async def deploy_and_relate_bundle_with_postgresql(
     await ops_test.model.wait_for_idle(
         apps=[application_name],
         status="active",
-        timeout=1000,
+        timeout=2000,
```

Comment:
This isn't really relevant to this ticket, but should adding a relation really take half an hour? It might be worth looking into why this takes so much time

Reply:
Good catch! I think this change was made just for a local test and I forgot to remove it. Reverted on https://github.com/canonical/postgresql-operator/pull/32/commits/0bb1414c7b9f6f9584826bedba08f5a61fc51f74.